### PR TITLE
JuliaRequirement: Add installation methods

### PR DESCRIPTION
**Pull request by Makman2**
The install_command method builds the command from its Julia attribute.

The is_installed method checks whether Julia is installed or not.

~~Fixes #2435~~

### CODE REVIEW

File: tests/bears/requirements/JuliaRequirementTest.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    def test_installed_requirement(self):
+        self.assertTrue(JuliaRequirement("Lint").is_installed())
+
+    def test_not_registered_requirement(self):
+        self.assertFalse(JuliaRequirement("some_bad_package").is_installed())
+
+    def test_registered_but_not_installed_requirement(self):
+        # FIXME Make this test more reliable, because `ACME` can be an existing
+        # FIXME package.
+        self.assertFalse(JuliaRequirement("ACME").is_installed())
```
Comment (SanketDG) [inline tests/bears/requirements/JuliaRequirementTest.py]: RANDOMRANDOM :smile:

Comment (SanketDG) [inline tests/bears/requirements/JuliaRequirementTest.py]: oh this is for not installed stuff..

File: tests/bears/requirements/JuliaRequirementTest.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import unittest
+import shutil
```
Comment (SanketDG) [inline tests/bears/requirements/JuliaRequirementTest.py]: needs sorting

File: tests/bears/requirements/JuliaRequirementTest.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+from coalib.bears.requirements.JuliaRequirement import JuliaRequirement
+
+
+@unittest.skipIf(shutil.which('julia') is None, "Julia is not installed.")
```
Comment (SanketDG) [inline tests/bears/requirements/JuliaRequirementTest.py]: Use single quotes here everywhere too?

File: .misc/deps.sh [hunk 4]
```diff
@@ -9,12 +9,20 @@ case $CIRCLE_NODE_INDEX in
...
+sudo add-apt-repository -y ppa:staticfloat/juliareleases
+sudo add-apt-repository -y ppa:staticfloat/julia-deps
 sudo apt-get update
 deps="indent libclang1-3.4 r-base"
 deps_python_dbus="libdbus-glib-1-dev libdbus-1-dev"
 deps_python_gi="glib2.0-dev gobject-introspection libgirepository1.0-dev python3-cairo-dev"
 deps_ruby_npm="gem nodejs"
-sudo apt-get install $deps $deps_python_gi $deps_python_dbus $deps_ruby_npm
+deps_julia="julia"
+deps_julia_packages="Lint"
```
Comment (SanketDG) [inline .misc/deps.sh]: move this downward after the apt get command

Comment (Makman2) [inline .misc/deps.sh]: hm you sure? I think having every dependency in this region of the file makes sense, then you can see every dependency at once without searching the whole doc imo^^

Comment (SanketDG) [inline .misc/deps.sh]: uh, so `julia` and `julia_deps` in successive lines, right, :+1: :smile:

Comment (SanketDG) [inline .misc/deps.sh]: but what I meant was only moving the line `deps_julia_packages="Lint"` to the part where they are actually installed.

Comment (Makman2) [inline .misc/deps.sh]: I think even deps_julia_packages belongs there, so you don't need to look around the file to find even those^^

**Comment by sils1297** [comment]
commit message: wait, this fixes a PR?!

**Comment by sils1297** [comment]
otherwise this looks good @Makman2 can you quockly fix the commit message?

**Comment by Makman2** [comment]
oh yes indeed, forgot to look at the message on cherry-picking :3

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala/pull/2732/commits/90b3efe78067c7dc176c11f2f63b74662442208f ack

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**